backends/ref/ceed-ref-operator.c

```diff
@@ -883,6 +883,229 @@ static int CeedOperatorApplyAddAtPoints_Ref(CeedOperator op, CeedVector in_vec,
   return CEED_ERROR_SUCCESS;
 }
 
+//------------------------------------------------------------------------------
+// Core code for assembling linear QFunction
+//------------------------------------------------------------------------------
+static inline int CeedOperatorLinearAssembleQFunctionAtPointsCore_Ref(CeedOperator op, bool build_objects, CeedVector *assembled,
```

Comment:
We may end up always having to `build_objects`, but that's something we can change in the future if needed

Reply:
I say leave the option for now, I think that we will be able to reuse the assembly objects at least between SNES iterations, since we only move the points after we converge (unless I'm forgetting something).